xapian-applications/omega/docs/omegascript.rst

```diff
@@ -222,6 +222,11 @@ $fmt
 $freq{term}
 	frequency of a term
 
+$hash{HASH,TEXT}
+    unique ID for TEXT string using the hashing algorithm specified by HASH.
+    Currently, this command only supports MD5 hashing algorithm which can be
+    used by specifying HASH as "md5" (without quotes).
```

Comment:
@jaylett @samuelharden Yes, it makes sense to make `HASH` optional with md5 as default. I've made the required changes.